tests/fixtures1.blocks/AnotherNamingBlock/Elem/AnotherNamingBlock-Elem.js

```diff
@@ -0,0 +1,16 @@
+import { decl } from 'bem-react-core';
+
+export default decl({
+    block : 'another-naming-block',
+    elem : 'elem',
+
+    mods : {
+        try : true
+    },
+
+    __dengerouslyNamingSettings : {
```

Comment:
shouldn't it be `__dangerouslySetNaming`?
also I confused that it's not belong to static fields